### PR TITLE
miniflux: 2.0.46 -> 2.0.47

### PR DESCRIPTION
**Pull request by pinpox**
## Things done
Updated version and hash

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/miniflux/default.nix [hunk 1]
```diff
@@ -11,7 +11,7 @@ in buildGoModule {
     owner = pname;
     repo = "v2";
     rev = version;
-    sha256 = "sha256-a27eKOhW2vHmPktLgqHKqiwtC9T6GRwnOeNReeMsaeM=";
+    sha256 = "sha256-SJw9C6s2JaI2AOVDazn1+N23Yp+Ul5P239tdAzXOxv4=";
   };
 
   vendorHash = "sha256-Oe7el4tE/gwI6qL/fjJgnv1jbNSKrCnq1nBq+dD7Gik=";
```
Comment (emilylange) [inline pkgs/servers/miniflux/default.nix:17]:
```suggestion
  vendorHash = "sha256-t1wFxXws3411/ZVNF6D4g7hDCSbmpCJhh8JCJ4QJdn4=";
```

**Review by emilylange — CHANGES_REQUESTED** [review]

File: pkgs/servers/miniflux/default.nix (hunk 1 above)
Comment (pinpox) [reply pkgs/servers/miniflux/default.nix]: Strange, I used `nix-update` but somehow got the hash wrong.

**Comment by sikmir** [comment]
Result of `nixpkgs-review pr 250948` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>miniflux</li>
  </ul>
</details>

**Comment by sikmir** [comment]
Could you please fix:
```diff
diff --git i/pkgs/servers/miniflux/default.nix w/pkgs/servers/miniflux/default.nix
index beaa6f9de914..8f2afcb2d6d1 100644
--- i/pkgs/servers/miniflux/default.nix
+++ w/pkgs/servers/miniflux/default.nix
@@ -21,7 +21,7 @@ in buildGoModule {
   checkFlags = [ "-skip=TestClient" ]; # skip client tests as they require network access

   ldflags = [
-    "-s" "-w" "-X miniflux.app/version.Version=${version}"
+    "-s" "-w" "-X miniflux.app/v2/internal/version.Version=${version}"
   ];

   postInstall = ''

```
See https://github.com/miniflux/v2/commit/168a870c025bfef6efdeb46e166e79a16093c157

**Comment by pinpox** [comment]
> ```diff
> -    "-s" "-w" "-X miniflux.app/version.Version=${version}"
> +    "-s" "-w" "-X miniflux.app/v2/internal/version.Version=${version}"
> ```

fixed

**Comment by emilylange** [comment]
Please squash your 4 commits into a single one titled `miniflux: 2.0.46 -> 2.0.47`.

See [`CONTRIBUTING.md#commit-conventions`](https://github.com/NixOS/nixpkgs/blob/8f90b41f081c34cf0631304b9e6fb0b986bfae45/CONTRIBUTING.md#commit-conventions)

**Comment by sikmir** [comment]
Superseded by https://github.com/NixOS/nixpkgs/pull/255439

**Pull request closed without merging**